Amazon.Ion.ObjectMapper/IonSerializer.cs

```diff
@@ -127,6 +129,8 @@ public class IonSerializationOptions
         public string[] AnnotatedTypeAssemblies { get; init; } = new string[] {};
 
         public readonly bool PermissiveMode;
+
+        public Dictionary<Type, dynamic> IonSerializers { get; init; }

```

Comment:
Would this make more sense as a `Dictionary<Type, IonSerializer<dynamic>>` (if possible)?